chunky/src/java/se/llbit/chunky/renderer/scene/Scene.java

```diff
@@ -1014,7 +1014,7 @@ public synchronized void loadChunks(TaskTracker taskTracker, World world, Collec
                 int cubeIndex = (cz * 16 + cy) * 16 + cx;
 
                 // Change the type of hidden blocks to ANY_TYPE
-                boolean notOnEdge = !chunkData.isBlockOnEdge(cx, cy, cz);
+                boolean notOnEdge = !chunkData.isBlockOnEdge(cx, y, cz) && !(y <= yMin || y >= yMax - 1);
                 boolean isHidden = notOnEdge
```

Comment:
```suggestion
                boolean onEdge = chunkData.isBlockOnEdge(cx, y, cz) || y <= yMin || y >= yMax - 1;
                boolean isHidden = !onEdge
```

I don't not think that this isn't not not harder to read.

Reply:
I was thinking the EXACT same thing, I didn't change it because `notOnEdge` has usages below, though I can just flip those too ;)

Reply:
Let's flip it then :+1: